src/rosdiscover/models/amcl.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-from ..interpreter import model
+# from ..interpreter import model
 
 M_PI = 3.14159265358979323846
 
 
-@model('amcl', 'amcl')
+# @model('amcl', 'amcl')
```

Comment:
?

Reply:
This is commented out so that we attempt to do recovery. Didn't want to delete in case we need it again.

Reply:
Done, with comment indicating why.